client/my-sites/preview/main.jsx

```diff
@@ -27,10 +30,28 @@ class PreviewMain extends React.Component {
 
 	state = {
 		previewUrl: null,
+		showingClose: false,
 	};
 
 	componentWillMount() {
 		this.updateUrl();
+		this.updateLayout();
+	}
+
+	updateLayout = () => {
+		this.setState( {
+			showingClose: isWithinBreakpoint( '<660px' ),
+		} );
+	}
+
+	throttledUpdateLayout = throttle( this.updateLayout, 100 );
```

Comment:
@spen I changed this over to `debounce` & halved the interval. It is fairly good at avoiding a suboptimal pop-in effect at that timing, but do you think we ought to go lower?